### ISSUE TITLE
Read-only files in character backup folder crashes dalamud

### ISSUE THREAD

**Issue by ff-meli** [goaaats/FFXIVQuickLauncher#138]
From SleepyBunBun in discord:

```System.UnauthorizedAccessException: Access to the path 'ACQ.DAT' is denied.
   at System.IO.Directory.DeleteHelper(String fullPath, String userPath, Boolean recursive, Boolean throwOnTopLevelDirectoryNotFound, WIN32_FIND_DATA& data)
   at System.IO.Directory.Delete(String fullPath, String userPath, Boolean recursive, Boolean checkHost)
   at XIVLauncher.Windows.MainWindow.<StartGame>d__14.MoveNext() in D:\Sapphire\FFXIVQuickLauncher\XIVLauncher\Windows\MainWindow.xaml.cs:line 400
4.5.0.0
86a8532
Context: Addons
Microsoft Windows NT 6.2.9200.0
True
True
True
True
True
English


[]
```

I thought the error might actually be the line before it mentions, where it tries to delete the old character backup folder - maybe one or more of the files is read-only for some reason.  But that should be caught, and the exception really does seem to be the addon iteration, and I didn't see anything there since the only deletion that happens with anything is on game close.

**Comment by ff-meli** [comment]
Apparently wiping the xivlauncher appdata folder and reinstalling fixed it.. but I'll leave this up in case it's worth looking at.

**Comment by ff-meli** [comment]
Wiping the appdata folder and reinstalling the launcher seemingly fixed it for one run, but then it started again.  Not entirely clear on behavior, but seemingly still a problem.

**Comment by ff-meli** [comment]
Sounds like manually deleting the character backup folder worked, at least for now.  While I don't know why any files in there would be read-only, the launcher should probably handle that case rather than crash out and fail to load addons and dalamud.

**Comment by Aida-Enna** [comment]
Fixed in #181
